modules/ossm-add-project-using-label-selectors-cli.adoc

```diff
@@ -21,10 +21,11 @@ You can use label selectors to add a project to the {SMProductShortName} with th
 +
 [source,terminal]
 ----
-$ oc edit smmr -n <controlplane_project>
+$ oc edit smmr default -n istio-system
 ----
 +
-The previous example uses `<controlplane_project>` as an example. You can deploy the {SMProductShortName} control plane to any project as long as it is separate from the project that contains your services.
+You can deploy the {SMProductShortName} control plane to any project as long as it is separate from the project that contains your services.
```

Comment:
Reword to: "...to any project provided that it is separate from the project that contains your services..